viite-UI/src/view/RoadAddressProjectEditForm.js

```diff
@@ -52,47 +53,70 @@
-        span = '<div class="project-edit-selections" style="display:inline-block;padding-left:8px;">' +
-          '<div class="project-edit">' +
-          ' TIE ' + '<span class="project-edit">' + link.roadNumber + '</span>' +
-          ' OSA ' + '<span class="project-edit">' + link.roadPartNumber + '</span>' +
-          ' AJR ' + '<span class="project-edit">' + link.trackCode + '</span>' +
-          ' M:  ' + '<span class="project-edit">' + startM + ' - ' + endM + '</span>' +
-          (selected.length > 1 ? ' (' + selected.length + ' linkkiä)' : '')+
-          '</div>' +
-          '</div>';
+        var div = '<div class="project-edit-selections" style="display:inline-block;padding-left:8px;">' +
+            '<div class="project-edit">' +
+            ' TIE ' + '<span class="project-edit">' + link.roadNumber + '</span>' +
+            ' OSA ' + '<span class="project-edit">' + link.roadPartNumber + '</span>' +
+            ' AJR ' + '<span class="project-edit">' + link.trackCode + '</span>' +
+            ' M:  ' + '<span class="project-edit">' + startM + ' - ' + endM + '</span>' +
+            (selected.length > 1 ? ' (' + selected.length + ' linkkiä)' : '')+
+            '</div>' +
+            '</div>';
+        span.push(div);
       }
       return span;
     };
 
+    var selectedSplitData = function (selected) {
+      var span = [];
+      _.each(selected, function(sel){
+        if (sel) {
+          var link = sel;
+          var startM = ((applicationModel.getSelectedTool() === 'Cut' && selected.length == 2) ? Math.round(link.startMValue) : Math.min.apply(Math, _.map(selected, function(l) { return l.startAddressM; })));
+          var endM = ((applicationModel.getSelectedTool() === 'Cut' && selected.length == 2) ? Math.round(link.endMValue) : Math.max.apply(Math, _.map(selected, function(l) { return l.endAddressM; })));
+          var div = '<div class="project-edit-selections" style="display:inline-block;padding-left:8px;">' +
+              '<div class="project-edit">' +
+              ' TIE ' + '<span class="project-edit">' + link.roadNumber + '</span>' +
+              ' OSA ' + '<span class="project-edit">' + link.roadPartNumber + '</span>' +
+              ' AJR ' + '<span class="project-edit">' + link.trackCode + '</span>' +
+              ' M:  ' + '<span class="project-edit">' + startM + ' - ' + endM + '</span>' +
+              '</div>' +
+              '</div>';
+          span.push(div);
+        }
+      });
+      return span;
+    };
+
     var defineOptionModifiers = function(option, selection) {
-      var roadIsUnknownOrOther = projectCollection.roadIsUnknown(selection[0]) || projectCollection.roadIsOther(selection[0]) || selection[0].roadLinkSource === 3;
+      var roadIsUnknownOrOther = projectCollection.roadIsUnknown(selection[0]) || projectCollection.roadIsOther(selection[0]);
+      var roadIsSuravage =  selection[0].roadLinkSource === 3;
```

Comment:
Is is possible to replace this _magic number_ with a constant or equivalent?

Reply:
All of those LinkSource and LinkStatus magic numbers should be together in a common Status.js or something.